src/arrayMethodReduce.js

```diff
@@ -5,7 +5,15 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    const initialValueTest = !(arguments.length < 2);
```

Comment:
Also, you can use just `!` operator 

```suggestion
    const initialValueTest = !!initialValue;
```

Reply:
This won't work when undefined or zero is passed as an initialValue.

Reply:
Too bad 🥲